File: lecture_002(Normal)/l_001_noofdigits.cpp
#include<iostream>
using namespace std;
int main(int args, char**argvs) {
    int num,count=0;
    cout<<"Enter a number:";
    cin>>num;
    while(num != 0) {

        num=num/10;
        count++;

    }
    cout<<"No.of digits:";
    cout<<count;
    
    return 0;
}

File: lecture_002(Normal)/l_001_noofdigits.java
import java.util.Scanner;
public class l_001_noofdigits {
    public static Scanner scn = new Scanner(System.in);
    public static void main(String[] args) {
        System.out.println("Enter a number:");
        int num = scn.nextInt();
        int count = 0;
        while(num != 0) {

            num=num/10;
            count++;

        }
        System.out.println("No. of digits:"+ count);
    }
}